clar2wasm/src/words/bindings.rs

```diff
@@ -18,6 +18,13 @@ impl ComplexWord for Let {
         _expr: &SymbolicExpression,
         args: &[SymbolicExpression],
     ) -> Result<(), GeneratorError> {
+        if args.len() < 2 {
+            return Err(GeneratorError::ArgumentLengthError(format!(
+                "let expected at least 2 arguments, got {}",
+                args.len()
+            )));
+        };
```

Comment:
Can you please cover these changes with tests?

Reply:
I’m updating them now accordingly to the review.

Reply:
Done, I've added the tests.